torchvision/prototype/transforms/_utils.py

```diff
@@ -17,6 +17,15 @@ def query_image(sample: Any) -> Union[PIL.Image.Image, torch.Tensor, features.Im
     raise TypeError("No image was found in the sample")
 
 
+def query_bboxes(sample: Any) -> features.BoundingBox:
```

Comment:
1. We are not querying boxes, since we only return one.
2. Let's keep the naming scheme consistent with the kernels.

```suggestion
def query_bounding_box(sample: Any) -> features.BoundingBox:
```